### PR TITLE
chore: bump containerd from 1.2.3 to 1.4.8

### PR DESCRIPTION
**Pull request by STRRL**
Signed-off-by: STRRL <str_ruiling@outlook.com>

close #2154

manually upgrade for https://github.com/chaos-mesh/chaos-mesh/pull/2154

### CODE REVIEW

**Comment by STRRL** [comment]
more vulnerabilities appear! let me just upgrade contaienrd to the latest version

**Comment by STRRL** [comment]
> more vulnerabilities appear! let me just upgrade contaienrd to the latest version

dependence conflicts happen! Just give up for a while.

I will retry that after upgrading the controller-runtime.

File: go.mod [hunk 1]
```diff
@@ -14,10 +14,11 @@ require (
 	github.com/chaos-mesh/chaos-mesh/api/v1alpha1 v0.0.0
 	github.com/chaos-mesh/k8s_dns_chaos v0.2.0
 	github.com/containerd/cgroups v1.0.2-0.20210605143700-23b51209bf7b
-	github.com/containerd/containerd v1.2.3
+	github.com/containerd/containerd v1.4.8
 	github.com/containerd/continuity v0.0.0-20200107194136-26c1120b8d41 // indirect
-	github.com/containerd/cri v1.11.1 // indirect
 	github.com/containerd/fifo v0.0.0-20191213151349-ff969a566b00 // indirect
+	github.com/containerd/go-runc v1.0.0 // indirect
+	github.com/containerd/ttrpc v1.0.2 // indirect
```
[the remark this answers is not in the document]
Comment (STRRL) [reply go.mod:21]: ignore

[the remark this answers is not in the document]
Comment (STRRL) [reply go.mod:21]: ignore

[the remark this answers is not in the document]
Comment (STRRL) [reply go.mod:21]: ignore

[the remark this answers is not in the document]
Comment (STRRL) [reply go.mod:21]: @sonatype-lift ignore

[the remark this answers is not in the document]
Comment (STRRL) [reply go.mod:21]: @sonatype-lift ignore

[the remark this answers is not in the document]
Comment (STRRL) [reply go.mod:21]: @muse-dev ignore

**Review by YangKeao — APPROVED** [review]
LGTM

**Review by fewdan — APPROVED** [review]
LGTM

**Pull request merged**